dev/mariadb/oldest/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   mariadb-oldest:
     container_name: sequelize-mariadb-oldest
-    image: mariadb:10.3.34
+    image: mariadb:10.4.6
```

Comment:
There is no good reason to support an old patch version, this should be 10.4.30

Reply:
Sure, I'll update that and make the changes to the test. The buffer is not needed anymore on recent patch versions

Reply:
Done